pkg/postage/service.go

```diff
@@ -47,32 +46,40 @@ type service struct {
 	store        storage.Store
 	postageStore Storer
 	chainID      int64
-	issuersInUse map[string]*semaphore.Weighted
+	issuers      []*StampIssuer
 }
 
 // NewService constructs a new Service.
-func NewService(store storage.Store, postageStore Storer, chainID int64) Service {
-	return &service{
+func NewService(store storage.Store, postageStore Storer, chainID int64) (Service, error) {
+	s := &service{
 		store:        store,
 		postageStore: postageStore,
 		chainID:      chainID,
-		issuersInUse: make(map[string]*semaphore.Weighted),
 	}
+	err := store.Iterate(
+		storage.Query{
+			Factory: func() storage.Item {
+				return new(StampIssuerItem)
+			},
+		}, func(result storage.Result) (bool, error) {
+			issuer := result.Entry.(*StampIssuerItem).Issuer
+			s.add(issuer)
```

Comment:
I think its better to write `_ = add()` to explicitly tell we are ignoring the return value.